README.md

```diff
@@ -19,7 +19,50 @@ This repository allows you to quickly install Apache Solr for Drupal 9+ into a [
      * Set "solr core" to `dev`
      * Under "Advanced server configuration" set the "solr.install.dir" to `/opt/solr`.
 
-1. `ddev restart`
+6. `ddev restart`
+
+## Installation on Silverstripe 4+
+
+1. `ddev get ddev/ddev-drupal9-solr && ddev restart`
+2. Install the required/relevant Silverstripe requirements: `ddev composer require firesphere/solr-search`
+    * Note that there are currently some tricky dependencies. This is known and to be fixed soon (tm)
+4. Solr is set to use the FileConfigStore. Ensure it's data location is at `.ddev/solr` and the host is set to `solr`:
+```yml
+---
+Name: LocalSolrSearch
+After:
+  - 'SolrSearch'
+Only:
+  environment: 'dev'
+---
+Firesphere\SolrSearch\Services\SolrCoreService:
+  config:
+    endpoint:
+      localhost:
+        host: solr
+  store:
+    path: './.ddev/solr'
+Firesphere\SolrSearch\Indexes\BaseIndex:
+  dev:
+    Classes:
+      - Page
+    FulltextFields:
+      - Title
+      - Content
+```
+5. Edit `.ddev/docker-compose.solr.yml` and on line 60, change `- ./solr:/solr-conf` to `- ./solr/dev:/solr-conf`
```

Comment:
If you're going to do this, it's critical to remove the #ddev-generated from the top of the file.

It would probably be better for you to fork this and maintain a silverstripe version of the solr add-on. 

With all the specialties here, you can see why this has "drupal9" in the title; your silverstripe instructions are as complicated as the Drupal ones.